core/jazz_acl/components/casbin.js

```diff
@@ -184,48 +189,130 @@ async function addPolicy(serviceId, policies, enforcer) {
 
 /* Get the permissions for a service given a userId */
 async function getPolicyForServiceUser(serviceId, userId, config) {
-  const result = await getPolicies(serviceId, config);
-  if (result && result.error) {
-    return result;
+  if (userId === config.SERVICE_USER) {
+    let result = attachAdminPolicies([serviceId]);
+    return result
+  } else {
+    const result = await getPolicies(serviceId, config);
+    if (result && result.error) {
+      return result;
+    }
+    let policies = formatPolicies(result);
+    let userPolicies = policies.filter(policy => policy.userId === userId);
+    userPolicies = userPolicies.map(policy => { return { permission: policy.permission, category: policy.category } });
+
+    return [{serviceId: serviceId, policies: userPolicies}];
+  }
+
+}
+
+/* fetch list of serviceIds from dynamodb */
+var scanResult;
+async function scanExecute(dynamodb, scanparams, items_formatted) {
+  let dataDb = await dynamodb.scan(scanparams).promise();
+
+  if (dataDb && dataDb.Items && dataDb.Items.length || items_formatted.length) {
+    dataDb.Items.forEach(function (item) {
+      items_formatted.push(item.SERVICE_ID.S);
+    });
+    if (dataDb.LastEvaluatedKey) {
+      scanparams.ExclusiveStartKey = dataDb.LastEvaluatedKey;
+      await scanExecute(dynamodb, scanparams, items_formatted);
+    } else {
+      scanResult = {
+        data : items_formatted
+      };
+    }
+  } else {
+    scanResult= {
+      error: dataDb || "db error"
+    };
   }
-  let policies = formatPolicies(result);
-  let userPolicies = policies.filter(policy => policy.userId === userId);
-  userPolicies = userPolicies.map(policy => { return { permission: policy.permission, category: policy.category } });
 
-  return [{serviceId: serviceId, policies: userPolicies}];
+  return scanResult
+}
+
+/* attach admin policies */
+function attachAdminPolicies(list) {
+  let svcIdList = []
+  list.forEach(eachId => {
+    let svcIdObj = {
```

Comment:
Read this from config  file, rather than hard coding here (Similar code already there)

Reply:
Updated